exchanges/coinbene/coinbene_wrapper.go

```diff
@@ -0,0 +1,369 @@
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (c *Coinbene) GetActiveOrders(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	var resp []exchange.OrderDetail
+	var tempResp exchange.OrderDetail
+	var tempData OpenOrderResponse
+	if len(getOrdersRequest.Currencies) == 0 {
+		allPairs, err := c.GetAllPairs()
+		if err != nil {
+			return resp, err
+		}
+		for a := range allPairs.Data {
+			getOrdersRequest.Currencies = append(getOrdersRequest.Currencies, currency.NewPairFromString(allPairs.Data[a].Symbol))
+		}
+	}
+	var err error
+	for x := range getOrdersRequest.Currencies {
+		tempData, err = c.FetchOpenOrders(exchange.FormatExchangeCurrency(c.Name, getOrdersRequest.Currencies[x]).String())
+		if err != nil {
+			return resp, err
+		}
+		for y := range tempData.OpenOrders {
+			tempResp.Exchange = c.Name
+			tempResp.CurrencyPair = getOrdersRequest.Currencies[x]
+			if tempData.OpenOrders[y].OrderType == buy {
+				tempResp.OrderSide = exchange.BuyOrderSide
+			}
+			if tempData.OpenOrders[y].OrderType == sell {
+				tempResp.OrderSide = exchange.SellOrderSide
+			}
+			tempResp.OrderDate = time.Unix(tempData.OpenOrders[y].OrderTime, 0)
+			tempResp.Status = tempData.OpenOrders[y].OrderStatus
+			tempResp.Price = tempData.OpenOrders[y].AvgPrice
+			tempResp.Amount = tempData.OpenOrders[y].Amount
+			tempResp.ExecutedAmount = tempData.OpenOrders[y].FilledAmount
+			tempResp.RemainingAmount = tempData.OpenOrders[y].Amount - tempData.OpenOrders[y].FilledAmount
+			tempResp.Fee = tempData.OpenOrders[y].TotalFee
+			resp = append(resp, tempResp)
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderHistory retrieves account order information
+// Can Limit response to specific order status
+func (c *Coinbene) GetOrderHistory(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	var resp []exchange.OrderDetail
+	var tempResp exchange.OrderDetail
+	var tempData ClosedOrderResponse
+	if len(getOrdersRequest.Currencies) == 0 {
+		allPairs, err := c.GetAllPairs()
+		if err != nil {
+			return resp, err
+		}
+		for a := range allPairs.Data {
+			getOrdersRequest.Currencies = append(getOrdersRequest.Currencies, currency.NewPairFromString(allPairs.Data[a].Symbol))
+		}
+	}
+	var err error
+	for x := range getOrdersRequest.Currencies {
+		tempData, err = c.FetchClosedOrders(exchange.FormatExchangeCurrency(c.Name, getOrdersRequest.Currencies[x]).String(), "")
+		if err != nil {
+			return resp, err
+		}
+		for y := range tempData.Data {
+			tempResp.Exchange = c.Name
+			tempResp.CurrencyPair = getOrdersRequest.Currencies[x]
+			if tempData.Data[y].OrderType == buy {
+				tempResp.OrderSide = exchange.BuyOrderSide
+			}
+			if tempData.Data[y].OrderType == sell {
+				tempResp.OrderSide = exchange.SellOrderSide
+			}
+			tempResp.OrderDate = time.Unix(tempData.Data[y].OrderTime, 0)
```

Comment:
According to the docs fetching orderids is returned like ` "orderTime": "2019-06-12T09:39:27.0Z"` and you don't do any conversion in the API implementation. This will die  🚶 💥🚗  You'll need to update your struct to use a string and then use `time.Parse(stringerino)` instead